README.md

```diff
@@ -47,8 +47,8 @@ Usage: token-airdrop (--mainnet | --testnet-magic NATURAL)
 - `asset-class`: Token asset class (overrides beneficiaries file config)
 - `drop-amount`: Amount of tokens to send to each beneficiary (overrides beneficiaries file config)
 - `beneficiaryPerTx`: This controls how many transaction outputs we batch together. In case the tranaction exceeds the size limit, try to change this value
-- `dryRun`: Builds transactions without actually submitting them on chain
-- `min-lovelaces`: Minimum lovelace amount for each utxo (change this it you get a Minimum required UTxO error)
+- `live`: Pass this flag to submit the transaction. Otherwise it is just printed to stdout
+- `min-lovelaces`: Minimun lovelace amount for each utxo (change this it you get a Minimum required UTxO error)
```

Comment:
Minimum is misspelled, I fixed it in my branch as I was modifying that area anyway; seems to have gotten reverted here